flyway-mysql57/Dockerfile.arm64

```diff
@@ -0,0 +1,24 @@
+FROM adoptopenjdk:8-jdk-hotspot-bionic
+
+LABEL version="5.7.21"
+LABEL maintainer="shinohara@chatwork.com"
+
+RUN apt-get update \
+    && apt-get install -y \
+        mysql-client-5.7 \
```

Comment:
@tshinowpub 
The version of the label is not specified.
You should always try to get the`5.7.21` version.

Reply:
@cw-ozaki Thank you!

The correct version was not included, so we fixed the version!

https://github.com/chatwork/dockerfiles/pull/1011/commits/f240fdce1b128e3365685dc3fc0e6c03335824f6#diff-b3b7a205b69151e303d5a14e557c31d97728b547140274553e20b0e1689c891bR13